lib/widgets/lightbox.dart

```diff
@@ -385,13 +385,14 @@ class _VideoPositionSliderControlState extends State<_VideoPositionSliderControl
 
   @override
   Widget build(BuildContext context) {
+    final zulipLocalizations = ZulipLocalizations.of(context);
     final currentPosition = _isSliderDragging
       ? _sliderValue
       : widget.controller.value.position;
 
     return Row(children: [
       VideoDurationLabel(currentPosition,
-        semanticsLabel: "Current position"),
+        semanticsLabel: zulipLocalizations.lightboxVideoCurrentPositionLabel),
```

Comment:
```suggestion
        semanticsLabel: zulipLocalizations.lightboxVideoCurrentPosition),
```

The "label" part doesn't really add any meaning here.